tests/python/contrib/test_cmsisnn/test_conv2d.py

```diff
@@ -70,7 +70,7 @@ def make_model(
     invar = relay.var("input", shape=shape, dtype=dtype)
     p = (0, 0, 0, 0)
     if padding == "INVALID":
-        p = [1, 2, 1, 2]
+        p = [1, 2, 2, 1]
```

Comment:
Given we've changed this case, do we have the previous behaviour captured in the non-matching test to ensure it's no longer matched?

Reply:
not exactly the same, but a similar one will capture the non-matching padding.